cloud/aws/variables.tf

```diff
@@ -6,6 +6,11 @@ variable "cockroach_port" {
   default = "26257"
 }
 
+# Comma-separated list of directories added as auxiliary stores.
+variable "aux_data_dirs" {
+  default = ""
```

Comment:
there's a default. It's an empty string, that means you get `data` and nothing else. I didn't want `data` itself to be configurable because that seemed like a hassle.